README.md

```diff
@@ -1,4 +1,4 @@
-# piet-gpu
+# vello (formerly piet-gpu)
```

Comment:
I'd rather we had the reference to `piet-gpu` within the first paragraph, but not directly in the title.

If we do want it in the title, I think we should set an end-date on it being the title; say 2023-03-09 as that is when Rust 1.68 is released; it also happens to be exactly 3 calendar months from now.